main.py

```diff
@@ -0,0 +1,466 @@
+import json
+import requests
+from requests.auth import HTTPBasicAuth
+
+
+#connect to kibana
+username = 'elastic'
+password = 'elastic'
```

Comment:
@Lucrezia-Cester  please don't put password in code

Reply:
@hshuaib90 makes sense, this is not the real password for kibana. I also slightly changed the info in the log so that it doesn't contain the correct identifiable info.

I tried authenticating to github with this code and it works (with proper username and password). Also I tested the sending to the http address and that also works. I guess the last piece is trying to get the logs from kibana directly?